packages/components/src/VirtualizedList/CellTextIcon/CellWithIcon.component.js

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Action from '../../Actions/Action';
+import styles from './CellWithIcon.scss';
+
+/**
+ * Cell renderer that displays a badge
+ */
+function CellWithIcon(props) {
```

Comment:
Please spread value in the function declaration so the alpi is more clear without diging into the code